package/gluon-config-mode-theme/sass/gluon.scss

```diff
@@ -71,13 +70,12 @@ html {
 }
 
 body, input, select, option {
-	font-family: 'Open Sans', Arial, sans-serif;
-	font-size: 12pt;
+	font: 12pt "Open Sans", Arial, sans-serif
```

Comment:
Missing semicolon

Reply:
you can actually remove the semicolon on the last attribute in {}.
but thats the only one in the complete file. so i will add a semicolon.

Reply:
fixed with f02b5f71278dfff70559172a2650b2c26e222945